.safety-policy.yml

```diff
@@ -19,6 +19,8 @@ security: # configuration for the `safety check` command
             reason: this only affects the --extra-index-url option in pip which we don't currently use. It is also intended behavior and is up to the user to operate --extra-index-url safely.
         70612:
             reason: we're only using jinja on the client side to template notebooks
+        71064:
+            reason: higher version requests cannot connect to docker
```

Comment:
In this file we want to record reasons for why the vulnerability reported isn't concerning for us. If this version fixed our bug but it opened a huge security hole, we'd figure out something else. 

[Here's a description of the vulnerability](https://data.safetycli.com/vulnerabilities/CVE-2024-35195/71064/). We never user `verify=False` so I'm ok ignoring this vulnerability. Can you change this to say something along those lines?